news/2018-04-12-new-featured-artist-dictate.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "New Featured Artist: Dictate"
+date: 2018-04-12 13:00:00 +0000
+---
+
+We're always searching for amazing talent to add to the Featured Artist lineup, and it turns out that we didn't have to look far from home - check out Dictate, the latest addition to the roster!
+
+[![](https://assets.ppy.sh/artists/21/header.jpg)](https://osu.ppy.sh/beatmaps/artists/21)
+
+No stranger to osu! with over five-hundred hours played, **[Dictate](https://osu.ppy.sh/beatmaps/artists/21)** is about as home-grown as Featured Artists get. When he's not clicking circles, he's creating the sounds that he wants to hear, and twisting them into gritty, neurofunk-inspired masterpieces.
+
+Hailing from **[Renraku Global Media's](https://soundcloud.com/renrakuglobalmedia)** lineup with a penchant for sound design and production that would easily rival someone ten years his senior, **Dictate**'s work sports an incredible range of rhythmic variance, underpinned with pounding and snarling bass, complex percussion and much more.
```

Comment:
don't understand the `someone ten years his senior` part

Reply:
someone ten years older than him, aka, he's very good for how old he is